files/conf/sshd_config

```diff
@@ -13,3 +13,4 @@ MaxAuthTries 5
 LoginGraceTime 60
 AllowTcpForwarding no
 AllowAgentForwarding no
+CASignatureAlgorithms ecdsa-sha2-nistp256,ecdsa-sha2-nistp384,ecdsa-sha2-nistp521,ssh-ed25519,rsa-sha2-512,rsa-sha2-256,ssh-rsa
```

Comment:
Assuming the others are just the defaults, so `ssh-rsa` is the only deviation?

Reply:
Correct , it's a list of default I found based on openbsd, ubuntu and debian.
- https://manpages.ubuntu.com/manpages/focal/en/man5/sshd_config.5.html
- https://man.openbsd.org/sshd_config.5#CASignatureAlgorithms
- https://manpages.debian.org/experimental/openssh-server/sshd_config.5.en.html

Arch linux specifies `sk` keys ( `sk-ssh-ed25519` and `sk-ecdsa-sha2-nistp256` ) which are for FIDO2 security keys.  Do you think it's worth adding those? 
- https://man.archlinux.org/man/ssh_config.5.en#CASignatureAlgorithms

Theoretically we could narrow it down even more to the list of keys used in employees repo and ssh-rsa.